sys/include/usb/usbus.h

```diff
@@ -195,16 +208,17 @@ typedef struct {
         /**
          * @brief USBUS generic header generator generated length
          *
-         * Must return the length of the header that will be generated by
-         * @ref get_header
+         * Must return the length total length of the descriptors that will be
+         * generated by @ref fmt_pre_descriptor and @ref get_header
          *
          * @param usbus The usbus context
          * @param arg   Additional argument for the header generator
          *
          * @return      Length of the generated header
          */
         size_t (*get_header_len)(usbus_t *usbus, void *arg);
-        size_t fixed_len;           /**< length of the header if it is a fixed length */
+        size_t fixed_len;           /**< total length of the generated
+                                         descripors if they are fixed length */
```

Comment:
```suggestion
                                         descriptors if they are fixed length */
```

Reply:
Wait, shouldn't have applied your suggestion, the fixed length must be the sum of the prefixed descriptor and the appended descriptor, so "descriptors" is correct here.

Reply:
Also clarified this one a bit

Reply:
what is bothering me here is the missing T

Reply:
> what is bothering me here is the missing T

I might have accidentally fixed that with my fixups